2018-09-10-was-ist-das-xcamp-und-wer-sollte-teilnehmen.md

```diff
@@ -0,0 +1,18 @@
+---
+title:     Video-Interview: Was ist das XCamp und wer sollte teilnehmen?
+author:    Jens Bothmer
+published: 10.09.2019
+image:     
+tags:      XCamp2019, Sessions
+---
+
+Interview mit Dr. Christiane Gerigk und Jens Bothmer – Initiatoren des Xcamps.
+
+Termin 13.09.2019
+Frankfurt am Main
+Skydeck Silberturm, 31. Etage
+bei DB Systel GmbH
+
+<a class="ytp-title-link yt-uix-sessionlink" target="_blank" data-sessionlink="feature=player-title" href="https://www.youtube.com/watch?v=4HbXMuR2eN0">Was ist das XCamp? Interview Dr. Christiane Gerigk und Jens Bothmer</a>
+
+<a class="button turquoise big" target="_self" href="tickets">JETZT TICKET SICHERN</a>
```

Comment:
We don't need capital letters here, we see them due to CSS in the "old" blog.

Reply:
I used normal writing style here